assets/javascripts/kitten/karl/pages/news-list/index.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import NewsCard from './news-card'
+import styled, { css } from 'styled-components'
+import { ArrowIcon } from '../../../components/icons/arrow-icon'
+import { ScreenConfig } from '../../../constants/screen-config'
+import { HorizontalStroke } from '../../../components/layout/horizontal-stroke'
+import { Title } from '../../../components/typography/title'
+import { Paragraph } from '../../../components/typography/paragraph'
+import { Text } from '../../../components/typography/text'
+import { Container } from '../../../components/grid/container'
+import { Button } from '../../../components/buttons/button/button'
+import { InstrumentTagIcon } from '../../../components/icons/instrument-tag-icon'
+import { pxToRem } from '../../../helpers/utils/typography'
+import COLORS from '../../../constants/colors-config'
+
+const Head = styled.div`
+  background-color: ${COLORS.primary5};
+  margin-bottom: ${pxToRem(100)};
+`
+
+const HeadContainer = styled(Container)`
+  height: 100px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const AnchorLink = styled(Text)`
+  padding: ${pxToRem(15)} ${pxToRem(30)};
+  ${props =>
+    props.selected &&
+    css`
+      color: ${COLORS.background1};
+      background-color: ${COLORS.primary1};
+      border-radius: ${pxToRem(4)};
+    `}
+`
+
+const BackLink = styled(Text)`
+  display: flex;
+  align-items: center;
+`
+
+const LeftArrowIcon = styled(ArrowIcon)`
+  fill: ${COLORS.primary1};
+  margin-right: ${pxToRem(5)};
+`
+
+const NewsContainer = styled.div`
+  padding: 0 ${pxToRem(20)};
+
+  @media (min-width: ${pxToRem(ScreenConfig.S.max)}) {
```

Comment:
je m'étais trompé dans le commentaire, voilà une version plus correcte

```suggestion
  @media (min-width: ${pxToRem(ScreenConfig.M.min)}) {
```